jerry-debugger/src/lib/cdt-controller.ts

```diff
@@ -52,8 +54,9 @@ export class CDTController {
   }
 
   onBreakpointHit(message: JerryMessageBreakpointHit) {
+    console.log('onBreakpointHit');
```

Comment:
Clean up?

Reply:
I did pull that out in the next PR... are we eventually going to squash them all or should I move the cleanup back here?